tests/safeds/ml/classical/regression/test_ridge_regression.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.ml.classical.regression import RidgeRegression
+
+
+def test_ridge_regression_invalid() -> None:
+    with pytest.raises(ValueError, match="The alpha parameter should not be below 0"):
+        RidgeRegression(alpha=-1.0)
+
+
+def test_ridge_regression_warning() -> None:
+    with pytest.warns():
```

Comment:
Should also include a `match`. Otherwise the test passes if _any_ warning is created. We want to ensure a specific warning is issued.